pkg/nodeconfig/init.go

```diff
@@ -17,7 +17,7 @@ type cache struct {
 	workerIgnitionEndPoint string
 }
 
-var log = logf.Log.WithName("nodeconfig")
+var log = ctrl.Log.WithName("nodeconfig")
```

Comment:
Shouldn't we be adding this logger to the node config struct too given this is the new pattern we are following?

Reply:
Done for both this and windows structs